test/e2e/operator_test.go

```diff
@@ -56,10 +56,10 @@ func TestRevisionLimits(t *testing.T) {
 
 	totalRevisionLimit := operatorSpec.SucceededRevisionLimit + operatorSpec.FailedRevisionLimit
 	if operatorSpec.SucceededRevisionLimit == 0 {
-		totalRevisionLimit += 5
+		totalRevisionLimit += 10
```

Comment:
For a follow-up: @damemi you wrote this original test. Why was this 5? Magic number?

Reply:
I now get the +=5 I believe (it's the default value?). But why does the loop below not wait for the revision pruning controller to do its work, but errors the test immediately?